### PR TITLE
Add evil80 keyboard

### PR DESCRIPTION
**Pull request by itsdrdick**

### CODE REVIEW

File: keyboards/evil80/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include "evil80.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* 0: Winkey */
+    LAYOUT(
+        KC_ESC,          KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,     KC_PSCR,KC_SLCK,KC_PAUS,\
+        KC_GRV,   KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,KC_MINS, KC_EQL,KC_BSPC,      KC_INS,KC_HOME,KC_PGUP,\
+        KC_TAB,   KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,KC_LBRC,KC_RBRC,KC_BSLS,      KC_DEL, KC_END,KC_PGDN,\
+       KC_CAPS,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,KC_SCLN,KC_QUOT,KC_NUHS, KC_ENT,                             \
+       KC_LSFT,KC_NUBS,   KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,KC_COMM, KC_DOT,KC_SLSH,KC_RSFT,   F(0),               KC_UP,        \
```
Comment (noroadsleft) [inline keyboards/evil80/keymaps/default/keymap.c]: Both instances of `F(0)` should be replaced with `MO(2)`, and then the `fn_actions` code block on Lines 33-35 should be deleted. That function is deprecated, and `MO(2)` will give equivalent functionality.

File: keyboards/evil80/rules.mk [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+#   Teensy halfKay   512
+#   Teensy++ halfKay 1024
+#   Atmel DFU loader 4096
+#   LUFA bootloader  4096
+#   USBaspLoader     2048
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
[the remark this answers is not in the document]
Comment (itsdrdick) [reply keyboards/evil80/rules.mk]: SMD chip and DFU loader

[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/evil80/rules.mk]: Okay, then this should be fine, then.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**